README.md

```diff
@@ -2,19 +2,26 @@
 
 In a number of different settings, interactions between clients and servers
 involve information that could be sensitive when associated with client
-identity.
-
-Client-server protocols like HTTP reveal aspects of client identity to servers
-through these interactions, especially source addresses.  Even without client
-identity, a server might be able to build a profile of client activity by
-correlating requests from the same client over time.
-
-In a setting where the information included in requests does not need to be
-correlated, the Oblivious HTTP protocol allows a server to accept requests via a
-proxy.  The proxy ensures that the server cannot see source addressing
+identity.  Client-server applications built on HTTP reveal aspects of client
+identity to servers through these interactions, especially source addresses.
+Even without client identity, a server might be able to build a profile of
+client activity by correlating requests from the same client over time.
+
+In HTTP-based applications where the information included in requests does not
+need to be correlated, the Oblivious HTTP protocol allows a supporting server to accept
+requests via a proxy.  The proxy ensures that the server cannot see source addressing
 information for clients, which prevents servers linking requests to the same
-client.  Encryption ensures that the proxy is unable to read requests or
-responses.
+client using such information.  Encryption ensures that the proxy is unable to read
+requests or responses.
+
+Example applications and use cases fit for the Oblivious HTTP protocol include DNS,
+data or telemetry submission, and special-purpose API calls such as Safe Browsing.
+In such application deployments, the relationship between client, server, and
```

Comment:
```suggestion
In some of these application deployments, the relationship between client, server, and
```
With revocation as an example, you could have a need to provide endpoints, so this isn't quite as strong as your original formulation.